content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md

```diff
@@ -0,0 +1,163 @@
+---
+title: >
+    End-User Q&A Series: Using OTel with GraphQL
+linkTitle: >
+    End-User Q&A: OTel with GraphQL
+date: 2023-02-07
+author: >-
+  [Adriana Villela](https://github.com/avillela) (Lightstep)
+  <br/> with contributions from: <br/>
+  [Rynn Mancuso](https://github.com/musingvirtual) (Honeycomb) |
+  [Reese Lee](https://github.com/reese-lee) (New Relic)
```

Comment:
Oh, ... could we find another way of acknowledging contributions? Maybe just putting it in the main text either at the end, or at the start (in smaller font and italics)?

Reply:
yeah, that looks odd, put it somewhere in an acknowledgment section or just list them as full authors, I did it that way in the past for some of my blog posts (the nginx apache ones)

Reply:
Fair point! I've added a line at the beginning of the post like this:

> *with contributions from: [Rynn Mancuso](https://github.com/musingvirtual) (Honeycomb) and [Reese Lee](https://github.com/reese-lee) (New Relic)*

Let me know if that works @svrnm @chalin?

Reply:
works for me.